### DISCUSSION TITLE
Branching Priorities for Discrete Variables in MIP?

### DISCUSSION THREAD

**Discussion by jeffreydeankelly2** [ERGO-Code/HiGHS#1541] [Q&A] [answered]
I am curious to know if discrete variable branching priorities are currently available or will be available in HiGHS' MIP in the future? - Thanks.

**Comment by jajhall** [comment, marked as answer]
Such priorities exist within the MIP solver but are not available to users. We currently have no plans to make them available

**Comment by jeffreydeankelly2** [comment]
OK, thanks.  They are accessible in other (commercial) MIP solvers and hence the question.

**Reply by jajhall** [reply]
Sure, and like many things that it would be nice to have, they could be made available in HiGHS. However, because we don't have the resources of a commercial software company, the priorities of other developments are such that we have no plan to add them.

**Comment by jeffreydeankelly2** [comment]
Makes sense - only asking the question :-)
